.gitignore

```diff
@@ -1 +1,7 @@
 vendor/
+
+###> friendsofphp/php-cs-fixer ###
+/.php_cs.cache
+###< friendsofphp/php-cs-fixer ###
+
+.idea/
```

Comment:
Remove `.idea` and add a new line at the end of file

🚨 Add this rule in your `.gitignore` global (https://help.github.com/articles/ignoring-files/#create-a-global-gitignore)